fixtures/TUFTestFixtureTerminatingDelegation/__init__.py

```diff
@@ -0,0 +1,20 @@
+from fixtures.builder import FixtureBuilder
+
+
+def build():
+    FixtureBuilder('TUFTestFixtureTerminatingDelegation')\
+        .publish(with_client=True)\
+        .create_target('targets.txt')\
+        .delegate('a', ['*.txt'])\
+        .create_target('a.txt', signing_role='a')\
+        .delegate('b', ['*.txt'], parent='a', terminating=True) \
+        .create_target('b.txt', signing_role='b') \
+        .delegate('c', ['*.txt'], parent='b') \
+        .create_target('c.txt', signing_role='c') \
+        .delegate('d', ['*.txt'], parent='b') \
+        .create_target('d.txt', signing_role='d') \
+        .delegate('e', ['*.txt'], parent='a') \
+        .create_target('e.txt', signing_role='e') \
+        .delegate('f', ['*.txt']) \
+        .create_target('f.txt', signing_role='f') \
+        .publish()
```

Comment:
@phenaproxima I haven't made the phpunit test that uses this fixture yet but I think this is how to make the test case specified in https://github.com/theupdateframework/specification/issues/168

I looked through the json files and I think it produces the correct fixture but let me know if you think it produces the correct fixture.

BTW I love how easy this is to make with the new `FixtureBuilder` class !